test/e2e/testing-manifests/flexvolume/attachable-with-long-format

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
```

Comment:
how about name it to attachable-with-long-mount?
Also each time we want to modify the volume plugin behavior, we have to create a new file with almost the same code?

Reply:
We can rename in a follow up PR, since this is very close to 1.13.  I will follow up this in - https://github.com/kubernetes/kubernetes/pull/71243 

What kind of behaviour modification to volume plugin you are talking about?  are you thinking of using flex to simulate some other edge cases in attach/detach flow?

Reply:
yes, it would be nice if we could simulate different cases on this volume plugin.